util/cmdgen.py

```diff
@@ -55,6 +56,7 @@ def docgen_rewrite_md(filepath: Path, dry_run: bool):
 
     '''
     content = filepath.read_text()
+    rel_path = filepath.relative_to(REPO_ROOT)
```

Comment:
Fixing the path in cmdgen is not mentioned in the commit message.

Reply:
This path is only used for log messages and doesn't effect functionality. I was adding log messages and thought the absolute paths were hurting readability.

Reply:
Ok, thanks